MonkeyBot/CommandManager.cs

```diff
@@ -155,12 +155,12 @@ private string GetCommandErrorMessage(CommandError error, string prefix, string
 
         public async Task BuildDocumentationAsync()
         {
-            string docuHTML = DocumentationBuilder.BuildDocumentation(commandService, DocumentationOutputTypes.HTML);
-            string fileHTML = Path.Combine(AppContext.BaseDirectory, "documentation.html");
+            var docuHTML = DocumentationBuilder.BuildDocumentation(commandService, DocumentationOutputTypes.HTML);
```

Comment:
I'd like to stick to the corefx coding style regarding var

> We only use var when it's obvious what the variable type is (e.g. var stream = new FileStream(...) not var stream = OpenStandardInput()).

Could you kindly either revert the commit or manually only assign var to places where the type will be obvious? I can also do it based on this commit of course

Reply:
I can revert the commit if you want?
Or is that what you said :P

Reply:
That would probably be the easiest way ;)

Reply:
And too answer your first question, yes i used ReSharper and some other small tools for this :)